utils.groovy

```diff
@@ -199,4 +199,28 @@ def get_push_trigger() {
     ]
 }
 
+def change_metal_artifacts_list_order(artifacts) {
+    if (artifacts.contains('live')) {
+        artifacts.remove("metal")
+        artifacts.remove("metal4k")
+        artifacts.remove("live")
+
+        artifacts.add(0, "metal")
+        artifacts.add(1, "metal4k")
+        artifacts.add("live")
+    }
+    return artifacts
+}
+
+def get_artifacts_to_build(pipecfg, stream, basearch) {
```

Comment:
add comment above this function? ``// gets desired artifacts to build from pipeline config`.